commit 4690124107fa99ff18bb77a1f19665edbfedd588
Author: Valentina Mukhamedzhanova <vmukhame@redhat.com>
Date:   Tue Apr 5 15:05:27 2016 +0200

    Clarify using 'group list' command for hidden groups in the man page.

diff --git a/docs/yum.8 b/docs/yum.8
index b837f9f..9c09c48 100644
--- a/docs/yum.8
+++ b/docs/yum.8
@@ -283,6 +283,8 @@ default package are installed (when not in group_command=objects mode).
 You can pass optional arguments to the list/summary commands: installed,
 available, environment, language, packages, hidden and ids (or any of those
 prefixed by "no" to turn them off again).
+Note that groups that are available but hidden will not be listed unless
+\'hidden\' keyword is passed to the command.
 If you pass the \-v option, to enable verbose mode, then the groupids are
 displayed by default (but "yum group list ids" is often easier to read).
 
